Dockerfile.assets

```diff
@@ -58,3 +23,12 @@ COPY --chown=elife:elife source/ ${PROJECT_FOLDER}/source
 
 COPY --chown=elife:elife assets/sass ${PROJECT_FOLDER}/assets/sass
 RUN gulp --environment ${environment} generateStyles
+
```

Comment:
shouldn't everything before this go into the `node` image? Then the `ci` image can use it (e.g. the `gulpfile.js` absence which the makes the build fail)